operations/research-and-development/product/analytics/looker/dashboards.md

```diff
@@ -10,15 +10,48 @@
 * [Customer Success Account Health Score](https://mattermost.looker.com/dashboards/36)
 * [Support Overview](https://mattermost.looker.com/dashboards/59)
 
+| Dashboard | Brief description |
+| ----------|-------------------|
+| [[Draft] List of self-hosted customers where telemetry is turned on](https://mattermost.looker.com/explore/data_warehouse/license_server_fact?qid=HcJOghkmxRtMf8Ui3eIE0s&toggle=fil) | Dashboard showing self-hosted customers with telemetry turned on (number of paid seats, number of activated users) 
```

Comment:
Propose saving as a Look, so that people don't need additional permissions to view the data (if kept in the Explore view, only users with Explore access will be able to view)